chart/kubeapps/templates/kubeapps-frontend-config.yaml

```
@@ -38,6 +38,9 @@ data:
         # Hide Www-Authenticate to prevent it triggering a basic auth prompt in
         # the browser with some clusters
         proxy_hide_header Www-Authenticate;
+        # Keep the connection open with the API server even if idle.
+        # Default 60 seconds, see https://github.com/kubeapps/kubeapps/issues/766
```

This comment is confusing, sounds like you're saying that we're defaulting it to 60s.

```suggestion
        # nginx defaults this to 60 seconds, we override it to 1h. See https://github.com/kubeapps/kubeapps/issues/766
```